linters/.eslintrc

```diff
@@ -126,5 +126,7 @@
     'no-with': 2,
     'max-len': [2, 120, 2, { 'ignoreUrls': true }],
     'use-isnan': 2,
+    // we want to explicitly define boolean values within jsx
+    'jsx-boolean-value': 0,
```

Comment:
`react` rules are name spaced, so you would want to indicate `react/jsx-boolean-value`

Also, we should enforce *always* defining a boolean, so rule should be `'react/jsx-boolean-value': [2, 'never'],`